integration_tests/tests/forseti/scripts/update_server_config.py

```diff
@@ -0,0 +1,85 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import subprocess
+import sys
+import yaml
+
+FORSETI_SERVER_CONFIG = '/home/ubuntu/forseti-security/configs/forseti_conf_server.yaml'
+
+
+class ServerConfigActions:
+
+    @staticmethod
+    def read_server_config():
+        try:
+            with open(FORSETI_SERVER_CONFIG, 'r') as stream:
+                config = yaml.safe_load(stream)
+            print('Successfully read server config...')
+            return config
+        except Exception as e:
+            print(f'ERROR: Unable to read server config. {e}')
+            sys.exit(-2)
+
+    @staticmethod
+    def reload_server_config():
+        try:
+            cmd = ['forseti', 'server', 'configuration', 'reload']
+            forseti = subprocess.run(cmd,
+                                     stderr=subprocess.PIPE,
+                                     stdout=subprocess.PIPE)
+            if forseti.returncode != 0:
+                print(f'ERROR: Forseti returned a nonzero exit code. {forseti.returncode}')
+                sys.exit(forseti.returncode)
+            print('Successfully reloaded server config...')
+        except Exception as e:
+            print(f'ERROR: Unable to reload server config. {e}')
+            sys.exit(-4)
+
+    @staticmethod
+    def set_cai_enabled(cai_enabled):
```

Comment:
Added this script because using the `tee` command adds a lot of noise to the output of the tests by dumping the entire config file. This can be used for other server config options (e.g. enable a scanner) to help make the test logs easier to read.